src/lib.rs

```diff
@@ -266,6 +275,43 @@ where
         }
     }
 
+    fn estimate_remaining_inner(info: &FsInfoReply) -> u32 {
+        let block_size = info.block_info.as_ref().map(|i| i.size).unwrap_or(255);
```

Comment:
does it really make sense to guess a block size?  I would prefer to just skip the estimation in that case and just use the hard limit.